renderers/offday.py

```diff
@@ -2,6 +2,9 @@
 from utils import get_font, center_text_position
 from data.data import Data
 import debug
+import time
+from datetime import datetime
+from time import strftime

```

Comment:
remove `import time` since you've already imported strftime then remove the datetime import since you're not using it anywhere